.github/workflows/fleet-and-orbit.yml

```diff
@@ -251,7 +252,7 @@ jobs:
     strategy:
       matrix:
         go-version: ['${{ vars.GO_VERSION }}']
-    runs-on: ubuntu-latest
+    runs-on: macos-12
```

Comment:
compiling for darwin arm64 requires CGO and a macOS runner because of this table: https://github.com/fleetdm/fleet/blob/tuf-universal/orbit/pkg/table/app-icons/app_icons_darwin.go